Database/Corrections/cataQuestFixes.lua

```diff
@@ -1214,26 +1221,50 @@ function CataQuestFixes.Load()
         [13708] = { -- Valiant Of Sen'jin
             [questKeys.requiredRaces] = raceKeys.ALL_HORDE,
         },
+        [13712] = { -- To The Rescue
+            [questKeys.extraObjectives] = {{nil,Questie.ICON_TYPE_TALK,l10n("Speak to Kadrak"),0,{{"monster",8582}}}},
```

Comment:
Same as above: new string that needs to be added to the translations and also `triggerEnd` instead of `objectives`.